pkg/apis/mattermost/v1alpha1/clusterinstallation_types.go

```diff
@@ -39,20 +48,33 @@ type DatabaseType struct {
 	ExternalDatabaseSecret string `json:"externalDatabaseSecret,omitempty"`
 }
 
+// RunningState is the state of the Mattermost instance
+type RunningState string
+
+const (
+	// Creating is the state when the Mattermost instance is being created
+	Creating RunningState = "creating"
+	// Upgrading is the state when the Mattermost instance is being upgraded
+	Upgrading RunningState = "upgrading"
+	// Running is the state when the Mattermost instance is fully running
+	Running RunningState = "running"
```

Comment:
Worth converging on `stable` to have consistent terminology with the provisioning server? (0/5).

Reply:
That's totally fine. This is a very rough stub implementation right now that I will be modifying quite a bit in my next PR to support upgrades. I will change this then.